consensus/vbft/chain_store.go

```diff
@@ -48,13 +72,50 @@ func (self *ChainStore) GetChainedBlockNum() uint32 {
 	return self.chainedBlockNum
 }
 
+func (self *ChainStore) SetExecMerkleRoot(blkNum uint32, merkleRoot common.Uint256) {
+	if blk, present := self.pendingBlocks[blkNum]; blk != nil && present {
```

Comment:
suggest to change to setExecMerkleRoot()

Reply:
has modify